### PR TITLE
[Rebranding] [Design system] Create updated typography page for testing and comparison

### PR DESCRIPTION
**Pull request by levadadenys**
### [Rebranding] [Design system] Create updated typography page for testing and comparison

This page is only needed for development purposes.

Once it'll be merged and deployed you'll be able to compare pages via links:
existing typography: https://code.org/rebrand/rebrand-type-spec
updated typography:  https://code.org/rebrand/upd-rebrand-type-spec

Screenshot: 
image

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/rebrand/upd-rebrand-type-spec.css [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+/* UPDATED TYPOGRAPHY STYLES STARTS HERE*/
+/**/
+/*----------------------------------------*/
+
+html {
+  font-size: 100%;
+}
+
+h1 {
+  font-size: 48px;
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/rebrand/upd-rebrand-type-spec.css:98]: Curious why all of the font sizes are hard-coded?

File: pegasus/sites.v3/code.org/public/rebrand/upd-rebrand-type-spec.css [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+  margin: 0;
+}
+
+h6 {
+  font-size: 16px;
+  line-height: 1.5rem;
+  margin: 0;
+}
+
+p.body-one {
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/rebrand/upd-rebrand-type-spec.css:134]: I don't see this anywhere on the updated type spec page, can you add it so we can see what it looks like?

File: pegasus/sites.v3/code.org/public/rebrand/upd-rebrand-type-spec.css (hunk 1 above)
Comment (levadadenys) [reply pegasus/sites.v3/code.org/public/rebrand/upd-rebrand-type-spec.css:98]: It's hardcoded just for the speed of demo implementation

File: pegasus/sites.v3/code.org/public/rebrand/upd-rebrand-type-spec.css (hunk 2 above)
Comment (levadadenys) [reply pegasus/sites.v3/code.org/public/rebrand/upd-rebrand-type-spec.css:134]: I've hardcoded all the typography elements that are present in updated typography. But the goal of demo was simply to compare current typography vs updated. Showing all the new typography was not a part of this PR, will be implemented later if needed (or when we aprove/finilize new typography), for now you can check all the elements in figma

Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/rebrand/upd-rebrand-type-spec.css:134]: Ok cool, makes sense! Can you link to where I can see it in Figma?

File: apps/src/templates/Button.jsx
Comment (kelbyhawn) [inline apps/src/templates/Button.jsx:1]: @levadadenys did you mean to have the Checkbox and Curriculum Catalog code included in this PR?

Comment (levadadenys) [reply apps/src/templates/Button.jsx:1]: Nope. Those are not intended and will be fixed

File: pegasus/sites.v3/code.org/public/rebrand/upd-rebrand-type-spec.css (hunk 2 above)
Comment (levadadenys) [reply pegasus/sites.v3/code.org/public/rebrand/upd-rebrand-type-spec.css:134]: DMed you link in slack

**Comment by levadadenys** [comment]
^^^ upd
cleaned up PR from extra changes

**Comment by levadadenys** [comment]
Looks like we don't need this anymore since we now have the Storybook (both local & public). Closing this PR.

**Pull request closed without merging**